configs/env.json

```diff
@@ -1,18 +1,25 @@
 {
     "download_dir": ".stone-cli/v0.1.0",
-    "url": "https://github.com/zksecurity/stone-cli/releases/download/v0.1.0-alpha/stone-cli-linux-x86-64.tar.gz",
+    "linux_x86_64_url": "https://github.com/zksecurity/stone-cli/releases/download/v0.1.0-alpha/stone-cli-linux-x86-64.tar.gz",
+    "macos_aarch64_url": "https://github.com/zksecurity/stone-cli/releases/download/v0.1.0-alpha/stone-cli-macos-aarch64.tar.gz",
```

Comment:
Is it possible that we could just point the artifacts of:

https://github.com/dipdup-io/stone-packaging

e.g. https://github.com/dipdup-io/stone-packaging/releases/tag/v2.0.1

Still with the sha256 sums.

Then:

1. We avoid maintaining the binaries
2. We make them responsible for "binary transparency", showing provenance of the artifacts.

Reply:
So I think it might be better to stick to the current implementation because
1. I'm not sure how much their [stone prover repo](https://github.com/baking-bad/stone-prover/commits/main/) will diverge from the original repo, so we want to be selectively adopting their binaries
2. Maintaining the binaries (i.e. updating to new versions) is low-cost work
3. Their release cadence may fall behind the stone prover releases, so we may have to build it ourselves in the future anyways (case in point, we may have to update to Stone v6 soon)